src/block/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-position: center;
+  }
+
+  &--0 .stars__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__reviews {
+    width: 100%;
+    height: 16px;
+
+    margin: 0;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+    text-align: right;
+    color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used here but it's not defined in this file. Ensure that it is defined or imported from another SCSS file.